test/utils.js

```diff
@@ -111,4 +111,27 @@
 			setCookie: setCookie
 		};
 	};
+
+	/**
+	 * Loads file contents via ajax GET request and passes it to callback function
+	 * @param {string} path Path to file
+	 * @param {function} callback Function that accepts two arguments:
+	 *                            first one - ajax error message,
+	 *                            second one - file contents.
+	 * @returns {boolean}
+	 */
```

Comment:
Since this code is sync can we use `loadFileSync` name and return the contents instead of using a callback? Also, these comments are unnecessary, the behavior van be inferred through the usage. Also, the boolean return is not being used, no need to implement that.